src/main/msp/msp_protocol.h

```diff
@@ -59,7 +59,7 @@
 #define MSP_PROTOCOL_VERSION                0
 
 #define API_VERSION_MAJOR                   1  // increment when major changes are made
-#define API_VERSION_MINOR                   36 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
+#define API_VERSION_MINOR                   37 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
```

Comment:
Hi martin,

I need a condition to use the CleanFlight Configurator to detect whether the current firmware supports the RunCam Split and, if supported, open a RunCam Split Tab in the CleanFlight Configurator.
The following is one of the code I modified CleanFlight Configurator, is through apiVersion to detect whether the firmware supports RunCam Split code:
in Features.js:
        if (semver.gte(CONFIG.apiVersion, "1.37.0")) {
            features.push(
                {bit: 30, group: 'other', name:'RUNCAM_SPLIT'}
            );
        }

in main.js
    if (features.isEnabled('RUNCAM_SPLIT')) {
        $('#tabs ul.mode-connected li.tab_runcam_split').show();
    } else {
        $('#tabs ul.mode-connected li.tab_runcam_split').hide();
    }

so, if I can't inc the API_VERSION_MINOR，I can not detect whether the firmware supports the runCam split in the CleanFlight Configurator. Do you have any suggestions for this check?